packages/workflow-core/src/lib/types.ts

```diff
@@ -29,15 +29,83 @@ export interface WorkflowContext {
   lockKey?: string;
 }
 
+export type JMESPathString = string;
+
+export interface ChildWorkflow {
+  name: string;
+  runtimeId: string;
+  definitionId: string;
+  definitionVersion: 1;
+  stateNames: Array<string>;
+  contextToCopy: JMESPathString;
```

Comment:
I would suggest keeping the same logic as the API plugin with the request transformer and response transformer. so we would have consistency api-wise